crates/cli/src/util.rs

```diff
@@ -7,9 +7,12 @@ use eyre::Result;
 use openvm_build::{get_in_scope_packages, get_workspace_packages};
 use openvm_sdk::{
     config::{AppConfig, SdkVmConfig},
-    fs::{read_agg_halo2_pk_from_file, read_agg_stark_pk_from_file},
+    fs::read_agg_stark_pk_from_file,
     keygen::AggProvingKey,
 };
+
+#[cfg(any(feature = "evm-prove", feature = "evm-verify"))]
+use openvm_sdk::fs::read_agg_halo2_pk_from_file;
```

Comment:
it's only used once, maybe just specify the path directly in the function call below